### ISSUE TITLE
Graph Traversal using BFS 

### ISSUE THREAD

**Issue by ShriyanshiSrivastava** [ShriyanshiSrivastava/Data_Structures_For_Hacktoberfest#73]
You can add a code that uses BFS to traverse a Graph.

**Comment by CRAZYGEEKS04** [comment]
i can do it.

**Comment by ShriyanshiSrivastava** [comment]
> i can do it.

Go ahead with this issue :)

**Comment by snehashukkla** [comment]
@ShriyanshiSrivastava please assign this to me, i can solve this issue

**Comment by dollygupta07** [comment]
Hi,
I would do it in python. Can you please assign this to me?

Thanks.

**Comment by ShriyanshiSrivastava** [comment]
@dollygupta07 Go ahead with the issue :)

**Comment by dollygupta07** [comment]
> > Hi, I had a doubt. Why my PR is not marked as 'hacktober-accepted' after being merged?
> 
> @dollygupta07 Your PR is merged directly to the repo which is already hactober accepted. We can either label it or merge it or do both. All will lead to same result. You can check your hactoberfest profile , your 1 PR must have beed added ;)

Yes, it's there in my  hacktoberfest profile. Thank you so much for the clarification.
